webservices/legal_docs/load_legal_docs.py

```diff
@@ -492,3 +492,76 @@ def remap_citations(archived_murs):
         # Include regulations and us_code citations in the re-map
         mur.citations = get_citations(map(lambda c: c['text'], list(mur.citations['regulations']) + list(mur.citations['us_code'])))
         yield mur
+
+def enable_stemming():
+    es = utils.get_elasticsearch_connection()
+    settings = {
+        "mappings": {
+            "_default_": {
+                "properties": {
+                    "no": {
+                        "type": "string",
+                        "index": "not_analyzed"
+                    },
+                    "text": {
+                        "type": "string",
+                        "analyzer": "english"
+                    },
+                    "name": {
+                        "type": "string",
+                        "analyzer": "english"
+                    },
+                    "description": {
+                        "type": "string",
+                        "analyzer": "english"
+                    },
+                    "summary": {
+                        "type": "string",
+                        "analyzer": "english"
+                    }
+                }
+            }
+        },
+        "settings": {
+            "analysis": {"analyzer": {"default": {"type": "english"}}}
+        }
+    }
+
+    docs_new = Index('docs_new', using=es)
+    if not docs_new.exists():
+        es.indices.create('docs_new', settings)
+
+    doc_types = ['statutes', 'regulations', 'advisory_opinions', 'murs']
+    original_count = []
+    backed_up_count = []
+    reindexed_count = []
+    for _type in doc_types:
+        query = {"query": {"match_all": {}}}
+        total = Search().using(es).query(Q('term', _type=_type)).execute().hits.total
+        original_count.append(total)
+        all_docs = elasticsearch.helpers.scan(es, query, scroll='1m', index='docs', doc_type=_type, size=500)
+        all_docs = list(Result(doc).to_dict() for doc in all_docs)
+        count, _ = elasticsearch.helpers.bulk(es, all_docs, index='docs_new',
+                    doc_type=_type, chunk_size=100, request_timeout=30)
+        backed_up_count.append(count)
+        docs_new.refresh()
+        print("Backed up %d %s." % (count, _type))
+
+    if original_count == backed_up_count and sum(original_count) > 0:
```

Comment:
For my understanding, under what circumstances would these counts be off? Wouldn't it be simpler to report the discrepancy as it happens, i.e. per _type?